frontend/src/metabase/public/containers/PublicAction/PublicActionLoader.tsx

```diff
@@ -62,7 +62,7 @@ function PublicActionLoader({ params, setErrorPage }: Props) {
   }, [action, params.uuid, setErrorPage]);
 
   return (
-    <EmbedFrame footerVariant="big">
+    <EmbedFrame footerVariant="large">
```

Comment:
Turns out components that use `_.compose` to apply multiple HoC can't be properly type-checked :/
We'd need to agree on a way to approach this

Reply:
You could do something like this at the export

`export default _.compose(connect(mapStateToProps), withRouter)(EmbedFrame) as React.ComponentType<OwnProps>;`

I suspect the reason it doesn't work is because of how it's designed. Even if the prop does works, it isn't totally right. The inner component receives other props from HoC too so actually the actually wrapped component should only expect `OwnProps`.

I see in a lot of places, TS even suggests passing props that should only be passed from other HoC e.g. redux, and entity loader, which sounds wrong.